src/modules/auth/auth.service.ts

```diff
@@ -41,7 +41,13 @@ export class AuthService {
     if (!isPasswordValid) {
       throw new UnauthorizedException('Invalid password');
```

Comment:
Same here, instead telling user Invalid password, we return username or password not match with 401 code